### ISSUE TITLE
Flake8-Tests fail on TravisCI

### ISSUE THREAD

**Issue by icemac** [zopefoundation/Products.PythonScripts#36]
See https://travis-ci.org/zopefoundation/Products.PythonScripts

**Comment by jugmac00** [comment]
Linting works locally, though... this is odd.

```
jugmac00@jugmac00-XPS-13-9370:~/Projects/Products.PythonScripts$ tox -e=lint
lint installed: entrypoints==0.3,flake8==3.7.7,flake8-blind-except==0.1.1,flake8-coding==1.3.2,flake8-commas==2.0.0,flake8-debugger==3.1.0,flake8-deprecated==1.3,flake8-html==0.4.0,flake8-isort==2.7.0,flake8-quotes==2.0.1,flake8-string-format==0.2.3,flake8-todo==0.7,isort==4.3.21,Jinja2==2.10.1,MarkupSafe==1.1.1,mccabe==0.6.1,pycodestyle==2.5.0,pyflakes==2.1.1,Pygments==2.4.2,testfixtures==6.10.0
lint run-test-pre: PYTHONHASHSEED='145083877'
lint run-test-pre: commands[0] | mkdir -p /home/jugmac00/Projects/Products.PythonScripts/parts/flake8
lint run-test: commands[0] | isort --check-only --diff --recursive /home/jugmac00/Projects/Products.PythonScripts/src setup.py
lint run-test: commands[1] | - flake8 --format=html src tests setup.py
lint run-test: commands[2] | flake8 src tests setup.py
___________________________________________________________________________ summary ___________________________________________________________________________
  lint: commands succeeded
  congratulations :)
```

**Comment by jugmac00** [comment]
I deleted the .tox folder and re-ran the linter locally, and now it also fails locally.

The spots the source code which fail are already annotated with something like `# NOQA: flake8: P101` - previously, the fix was to turn it in something like `# noqa: P101`.

Maybe flake8 changed the discovery/noqa syntax recently? @asottile

**Comment by icemac** [comment]
Fixed in #37

**Comment by asottile** [comment]
flake8's noqa syntax was recently extended to support multi-character codes (`# noqa: ABC123`).  your `# noqa: flake8` is trying to silence a code named `FLAKE8` in the new syntax, in the old syntax it was silencing *all* codes
